.github/workflows/Nightly_Perf_Env_IPI_CI.yml

```diff
@@ -2,12 +2,12 @@
 # https://crontab.guru/
 # This is a nightly CI Pipeline against Performance environment using IPI installer - run on Sunday
 name: Deploy IPI Perf Env Nightly CI
-#on:
-#  schedule:
-#    - cron: '0 4 * * 0' # run on Sunday at 4 AM UTC/ 0 AM EST
 on:
-  push:
-    branches: [ main ]
+  schedule:
+    - cron: '0 4 * * 0' # run on Sunday at 4 AM UTC/ 0 AM EST
```

Comment:
Nit: 4 AM UTC is 11 PM EST or midnight EDT.  Just note that it's 4 AM UTC.

Reply:
midnight EDT.

Reply:
US/Eastern *Standard* Time is 5 hours behind UTC.  Eastern *Daylight* Time is 4 hours behind UTC:

```
$ date
Mon 08 Nov 2021 10:24:57 AM EST
$ date -u
Mon 08 Nov 2021 03:24:59 PM UTC
```

Reply:
OK I will change the comment like this in all the nightly CI:
    - cron: '0 4 * * *' # run at 4 AM UTC/ 0 AM EDT